src/main/java/core/basesyntax/service/impl/BalanceTransactionHandler.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.service.FruitServiceImpl;
+
+public class BalanceTransactionHandler implements TransactionHandler {
+    @Override
+    public void addTransaction(FruitTransaction fruitTransaction) {
+        FruitService fruitService = new FruitServiceImpl();

```

Comment:
Do not create FruitService inside method - make it a class field and use constructor to inject it.